File: Extra/Leseras/Estructura Lenguajes/Java/Manejo de Archivos.java
//En Java, el manejo de archivos es bastante flexible.
//Aquí tienes un ejemplo simplificado:

import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;

public class ManejoArchivos {
    public static void main(String[] args) {
        try (BufferedReader br = new BufferedReader(new FileReader("archivo.txt"))) {
            String linea;
            while ((linea = br.readLine()) != null) {
                System.out.println(linea);
            }
        } catch (IOException e) {
            System.out.println("Error al leer el archivo.");
        }
    }
}

//En Java, se utiliza un bloque try-with-resources para asegurarse de que los recursos (en este caso, el archivo) se cierren automáticamente después de su uso.

File: Extra/Leseras/Estructura Lenguajes/Python/Manejo de Archivos.py
# Escritura en un archivo
with open("archivo.txt", "w") as archivo:
    archivo.write("Hola, archivo!")

# Lectura de un archivo
with open("archivo.txt", "r") as archivo:
    contenido = archivo.read()
    print(contenido)

#La gestión de archivos se realiza mediante el uso de las funciones open() y close().
#El uso de with garantiza que el archivo se cierre automáticamente después de su uso.
#write() se utiliza para escribir en un archivo, mientras que read() se utiliza para leer el contenido del archivo.
#Espero que estos ejemplos detallados te ayuden a comprender mejor la sintaxis y las estructuras básicas en Python.